download/index.md

```diff
@@ -2,17 +2,17 @@
 layout: download
 title: Download
 
-scala2tutorials:
-- title: "Getting Started with Scala 2"
-  url: "https://docs.scala-lang.org/getting-started/index.html"
-- title: "Scala 2 documentation"
-  url: "https://docs.scala-lang.org/"
-
-scala3tutorials:
-- title: "Getting Started with Scala 3"
-  url: "https://docs.scala-lang.org/scala3/getting-started.html"
+advancedtutorials:
 - title: "Scala 3 documentation"
   url: "https://docs.scala-lang.org/scala3/"
 - title: "Migrating from Scala 2 to Scala 3"
   url: "https://docs.scala-lang.org/scala3/guides/migration/compatibility-intro.html"
+
+newcomertutorials:
+- title: "Scala Book"
+  url: "https://docs.scala-lang.org/scala3/book/introduction.html"
+- title: "Tour of Scala"
+  url: "https://docs.scala-lang.org/tour/tour-of-scala.html"
```

Comment:
It is a bit confusing that here we mix links to Scala 3 resources (book, online courses) and Scala 2 resources (tour). I believe we should plan to update the tour to Scala 3 soon, maybe this is already what you wanted to do?

Reply:
I think it will be a bit of a mess until the structure of the documentation is finalised, so we could revisit at that point